specification/confidentialledger/data-plane/Microsoft.ConfidentialLedger/preview/2024-12-09-preview/confidentialledger.json

```diff
@@ -814,6 +814,150 @@
         }
       }
     },
+    "/app/userDefinedFunctions": {
```

Comment:
Missing functionName path parameter.

Reply:
Agreed. Moved the put operation under the endpoint `/app/userDefinedFunctions/{functionId}` below and adjusted the properties accordingly.